opencti-platform/opencti-front/src/private/components/settings/support/SupportPackageLine.tsx

```diff
@@ -0,0 +1,280 @@
+import React, { FunctionComponent, useState } from 'react';
+import { graphql, useFragment, useMutation } from 'react-relay';
+import { SupportPackageLine_node$key } from '@components/settings/support/__generated__/SupportPackageLine_node.graphql';
+import ListItemText from '@mui/material/ListItemText';
+import makeStyles from '@mui/styles/makeStyles';
+import ListItem from '@mui/material/ListItem';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import { FileOutline } from 'mdi-material-ui';
+import ListItemSecondaryAction from '@mui/material/ListItemSecondaryAction';
+import IconButton from '@mui/material/IconButton';
+import { DeleteOutlined, DownloadingOutlined, GetAppOutlined } from '@mui/icons-material';
+import Tooltip from '@mui/material/Tooltip';
+import DialogContent from '@mui/material/DialogContent';
+import DialogContentText from '@mui/material/DialogContentText';
+import DialogActions from '@mui/material/DialogActions';
+import Button from '@mui/material/Button';
+import Dialog from '@mui/material/Dialog';
+import { RecordSourceSelectorProxy } from 'relay-runtime';
+import CircularProgress from '@mui/material/CircularProgress';
+import Chip from '@mui/material/Chip';
+import { APP_BASE_PATH, handleError, MESSAGING$ } from '../../../../relay/environment';
+import { useFormatter } from '../../../../components/i18n';
+import Transition from '../../../../components/Transition';
+import { deleteNode } from '../../../../utils/store';
+import { hexToRGB } from '../../../../utils/Colors';
+import { DataColumns } from '../../../../components/list_lines';
+
+const useStyles = makeStyles(() => ({
+  bodyItem: {
+    height: 20,
+    fontSize: 13,
+    float: 'left',
+    whiteSpace: 'nowrap',
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    paddingRight: 10,
+  },
+  item: {
+    paddingLeft: 10,
+    height: 50,
+    cursor: 'default',
+  },
+  chipInList: {
+    fontSize: 12,
+    height: 20,
+    float: 'left',
+    borderRadius: 4,
+    width: 120,
+  },
+  label: {
+    whiteSpace: 'nowrap',
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+  },
+}));
```

Comment:
`makeStyles` is deprecated, please put your style directly in the JSX with props `sx` and `style`